build/ci.go

```diff
@@ -390,6 +459,62 @@ func downloadLinter(cachedir string) string {
 	return filepath.Join(cachedir, base, "golangci-lint")
 }
 
+var errNoReleaseAvailable = errors.New("no release available")
+
+// downloadProtoc downloads the prebuilt protoc binary used to lint generated
+// files as a CI step.  It returns the full path of the 'protoc' executable or
+// an error.
+func downloadProtoc(cachedir string) (string, error) {
+	version := "27.1"
```

Comment:
We have a method to retrieve the version from the ChecksumDB, please use it here.